### PR TITLE
Add number of authors and number of revisions to entity API

### PR DESCRIPTION
**Pull request by inyono**
Not sure whether this is exactly the needed use case: Added the wanted information to the entity API I already know from the mobile app (i.e. endpoint: `entities/:id`)

### CODE REVIEW

**Comment by arekkas** [comment]
Could you rename `countAuthors` to `authorsCount` or `numOfAuthors`? Same thing for revisions please :)

**Comment by arekkas** [comment]
Otherwise this looks good, I *believe* that this is what Knewton asked for.

**Comment by SimonKoehl** [comment]
They asked for the listed output of all articles, all exercises etc.. Aeneas, can you send us these URLs please, I can't find them anymore?

**Comment by inyono** [comment]
This one maybe: `https://de.serlo.org/entity/api/json/export/:type`?

**Comment by arekkas** [comment]
https://de.serlo.org/entity/api/json/export:{{type}}

**Comment by arekkas** [comment]
aka `article`, `course`, `course-page`, ...

**Comment by arekkas** [comment]
...might not work for the english version, please make sure to test that first

**Comment by arekkas** [comment]
By the way, the articles etc don't have content, only a content teaser

File: src/module/Entity/src/Entity/Controller/ApiController.php [hunk 1]
```diff
@@ -184,7 +194,9 @@ protected function normalize(Collection $collection)
                 'keywords'     => $normalized->getMetadata()->getKeywords(),
                 'categories'   => $this->getCategories($entity),
                 'link'         => $this->url()->fromRoute($normalized->getRouteName(), $normalized->getRouteParams()),
-                'lastModified' => $normalized->getMetadata()->getLastModified()
+                'lastModified' => $normalized->getMetadata()->getLastModified(),
+                'authorsCount' => count(array_unique($authors->toArray())),
```
Comment (arekkas) [inline src/module/Entity/src/Entity/Controller/ApiController.php]: RSS doesn't support these, so no need for that

**Comment by inyono** [comment]
Okay, I also added these information to `/entity/api/json/export/:type`.

> ...might not work for the english version, please make sure to test that first

It does, at least live.

File: src/module/Entity/src/Entity/Controller/ApiController.php (hunk 1 above)
Comment (arekkas) [inline src/module/Entity/src/Entity/Controller/ApiController.php]: Oh sorry nevermind, this is used in export as well...

**Comment by arekkas** [comment]
> It does, at least live.

It's great to see multi tenancy working :D

File: src/module/Entity/src/Entity/Controller/ApiController.php (hunk 1 above)
Comment (inyono) [inline src/module/Entity/src/Entity/Controller/ApiController.php]: Is this a problem, i.e. do I have to differentiate between export and rss?

**Comment by arekkas** [comment]
I'll deploy this probably tonight, maybe tomorrow

**Comment by SimonKoehl** [comment]
If I use https://de.serlo.org/entity/api/json/export/:article, I get a "serious error".. Am I doing something wrong?

**Comment by arekkas** [comment]
Don't put a `:` in front then ;)

**Comment by SimonKoehl** [comment]
thanks!

Schould we split "categories" in "topic" and "curricula"?

**Comment by SimonKoehl** [comment]
And while we are on this right now. Do you have any further ideas for valuable information, we should include?

**Comment by SimonKoehl** [comment]
@inyono @arekkas 

Schould we split "categories" in "topic" and "curricula", so it might be easier for Elixier to identify and map Serlo content with official curricula and official topic/taxonomy-metadata?

**Comment by arekkas** [comment]
has a little more work into it, let's discuss this at a different time and see if people actually need it!

**Comment by SimonKoehl** [comment]
all right!

**Pull request merged**